contrib/pg_tde/src/catalog/tde_principal_key.c

```diff
@@ -706,6 +706,7 @@ pg_tde_delete_key(PG_FUNCTION_ARGS)
 		if (default_principal_key == NULL)
 		{
 			ereport(ERROR,
+					errcode(ERRCODE_OBJECT_IN_USE),
 					errmsg("cannot delete principal key"),
```

Comment:
Should this also get "which is currently in use"?

Reply:
Seems like that is not a common PG way of writing it, so probably not. But maybe it should still be improved.